extensions/vertx-web/deployment/src/main/java/io/quarkus/vertx/web/deployment/Methods.java

```diff
@@ -168,6 +173,17 @@
     static final MethodDescriptor OPTIONAL_OF_NULLABLE = MethodDescriptor
             .ofMethod(Optional.class, "ofNullable", Optional.class, Object.class);
 
+    static final MethodDescriptor GET_VALIDATOR = MethodDescriptor.ofMethod(ValidationSupport.class, "getValidator",
```

Comment:
It's good to start the name of a constant with the class/domain name, e.g. `VALIDATION_GET_VALIDATOR`, `SET_IS_EMPTY`...

Reply:
Fixed.